src/byu_cas/core.clj

```diff
@@ -13,7 +13,7 @@
 
 (defn pprints-mw [handler]
   (fn [req]
-    (println "req:" )
+    (println "req:")
```

Comment:
No println in production code. If you must have it, it needs to be a timbre/log statement

Reply:
Those are in the current master branch (just flagged here because of whitespace change). Should I remove all the printlns from the entire core.clj?